build.sbt

```diff
@@ -19,16 +19,18 @@ playVersion := "3.0.1"
 
 libraryDependencies ++= Seq(
   // play framework cache API
-  "org.playframework"   %% "play-cache"                % playVersion.value % Provided,
+  "org.playframework" %% "play-cache" % playVersion.value % Provided,
   // redis connector
-  "io.github.rediscala" %% "rediscala"                 % "1.14.0-pekko",
+  "io.lettuce" % "lettuce-core" % "6.3.2.RELEASE",
+  //Subsequent versions can attempt to remove it
+  "io.github.rediscala" %% "rediscala" % "1.14.0-pekko",
```

Comment:
Could you elaborate more on why which use is left to remove? Which code wasn't migrated?

Reply:
I have other services using port 7000 locally, you can restore it

Reply:
There are some dependencies related to spec testing that I have not made any changes to at the moment. After the subsequent changes are completed, I suggest deleting Redisscala because he is already dead